client/src/pages/login/judgingDash/judging.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import './judging.css';
+
+function Judging() {
+  return (
+  	<div className="App-judging" id="Judging">   {/* This code basically puts "Judging Dash"*/}
+	  	<div className="container">				 {/* in the middle of the page */}
+		  	<div className="judging">
+		  	<h1>Judging Dash</h1>   {/* Big word "Judging Dash" */}
+		  	</div>
+
+	  	</div>
+  	</div>
+  );
+}
+export default Judging;
+
+
+
```

Comment:
Extra returns at the end of a file should be linted out. I'm sure you can find a good extension for that on VS Code!

Reply:
Linted all the files. :)